src/App.tsx

```diff
@@ -114,11 +145,50 @@ function App() {
         );
         setOrdinalsAddress(ordinalsAddressItem?.address);
         setOrdinalsPublicKey(ordinalsAddressItem?.publicKey);
+
+        const stacksAddressItem = response.addresses.find(
+          (address) => address.purpose === AddressPurpose.Stacks
+        );
+        setStacksAddress(stacksAddressItem?.address);
+        setStacksPublicKey(stacksAddressItem?.publicKey);
       },
       onCancel: () => alert("Request canceled"),
     });
   };
 
+  const onConnectRPCClick = async () => {
```

Comment:
Looks like this won't set the stacks address, which means that isReady above won't be set to true in this scenario. I think once we have the btc addresses, we should do a second RPC call for the stacks address maybe?

Reply:
Removed the callback, since it's not currently being used.